From 2c721018cc3351ee4518f4f57bbf5b646f96d533 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Thu, 20 Feb 2020 18:58:27 +0100
Subject: [PATCH 1/1] xlat: update IPPROTO_* constants

* xlat/inet_protocols.in (IPPROTO_MPTCP): New constant, introduced
by Linux kernel commit v5.6-rc1~151^2~118^2~8.

Signed-off-by: Andrei Gherzan <andrei.gherzan@huawei.com>
Upstream-status: Backport
---
 xlat/inet_protocols.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xlat/inet_protocols.in b/xlat/inet_protocols.in
index 7e14dc058..5c27821a3 100644
--- a/xlat/inet_protocols.in
+++ b/xlat/inet_protocols.in
@@ -32,3 +32,4 @@ IPPROTO_UDPLITE		136
 IPPROTO_MPLS		137
 IPPROTO_RAW		255
 IPPROTO_MAX		256
+IPPROTO_MPTCP		262
-- 
2.33.0

